jupyter_foundation.md

```diff
@@ -0,0 +1,27 @@
+# Jupyter Foundation
+
+The Jupyter Foundation is a directed fund of the Linux Foundation 501(c)(6). It exists to provide resources and strategic counsel to Project Jupyter. The charter is here (TODO: Link). We summarize relevant points from the charter below.
+
+
+## Jupyter Foundation Governing Board
+
+The Jupyter Foundation Governing Board controls the resources of the Jupyter Foundation and works to build a healthy collaboration between the Jupyter community, Jupyter leadership, and members of the Jupyter Foundation, including for-profit organizations. The current members of the Jupyter Foundation Governing Board are listed in the [Leadership Directory](people.md#jupyter-foundation-governing-board).
+
+## Responsibilities of the Jupyter Foundation Governing Board
+
+- Serve Project Jupyter and its mission.
+- Meet regularly to review strategic objectives of Jupyter.
+- Develop and approve an annual strategic plan and budget for funding from the Jupyter Foundation to benefit Project Jupyter.
+- Recruit and retain Jupyter Foundation members.
+
+## Membership
+
+The Jupyter Foundation Governing Board consists of:
+
+- All members of the [Jupyter Executive Council](./executive_council.md)
+- One representative from each Jupyter Foundation Premier Member
+- No more than two representatives total from all Jupyter Foundation General Members
+
+## Decision making
+
+The Jupyter Foundation Governing Board follows the decision-making policies in its charter. (TODO: Link)
```

Comment:
To resolve any ambiguities here between our decision making policies and those of the LF, in addition to providing a link, we should clarify that this is the "Jupyter Decision Making policy".

Reply:
Other than adding the links (coming soon) we think this is clear enough - its charter spells out its specific decision making process, which is fairly straightforward but has more formal voting requirements in order to keep a clear record. I think it's unambiguous and we don't need to repeat it here, people can read the doc.

Reply:
I've clarified in a new commit here that the Jupyter Foundation Governing Board follows the decision-making guidelines in the Jupyter Foundation charter.